packages/manager/apps/veeam-backup/src/pages/dashboard/SubscriptionTile.component.tsx

```diff
@@ -46,13 +51,10 @@ export const SubscriptionTile: React.FC<VeeamBackupWithIam> = ({
             <OsdsSkeleton />
           ) : (
             <>
-              {data?.data?.billing?.nextBillingDate && (
-                <Description className="block">
-                  {formatDateString(
-                    data.data.billing.nextBillingDate,
-                    ovhLocaleToI18next(i18n.language),
-                  )}
-                </Description>
+              {nextBillingDate ? (
+                <Description className="block">{nextBillingDate}</Description>
+              ) : (
+                <OsdsSkeleton />
```

Comment:
here, `isLoading` should be false so why display a skeleton ?

Reply:
Yes right
I fixed it